src/amqp-client/channel.cr

```diff
@@ -14,6 +14,7 @@ class AMQP::Client
     @confirms = ::Channel(Frame::Basic::Ack | Frame::Basic::Nack).new(1024)
     @next_msg_ready = ::Channel(Nil).new
     @log : Logger
+    @flow = true
```

Comment:
default true?

Reply:
Yeah, flow active is default state. Ticked me, no flow = don't send. flow = all good

> When a new channel is opened, it is active (flow is active) ...

https://www.rabbitmq.com/amqp-0-9-1-reference.html#channel.flow